exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/agg/TestAggWithAnyValue.java

```diff
@@ -18,62 +18,117 @@
 
 package org.apache.drill.exec.physical.impl.agg;
 
+import com.google.common.collect.Lists;
+import org.apache.drill.exec.physical.config.StreamingAggregate;
+import org.apache.drill.exec.physical.unit.PhysicalOpUnitTestBase;
 import org.apache.drill.test.BaseTestQuery;
 import org.apache.drill.categories.OperatorTest;
+import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 
 import org.apache.drill.test.TestBuilder;
+import org.junit.experimental.runners.Enclosed;
+import org.junit.runner.RunWith;
 
 import java.math.BigDecimal;
+import java.util.List;
 
 @Category(OperatorTest.class)
-public class TestAggWithAnyValue extends BaseTestQuery{
+@RunWith(Enclosed.class)
+public class TestAggWithAnyValue {
 
-  private static final String disableHashAgg = "alter session set `planner.enable_hashagg` = false";
-  private static final String defaultHashAgg = "alter session set `planner.enable_hashagg` = true";
+  public static class TestAggWithAnyValueMultipleBatches extends PhysicalOpUnitTestBase {
 
-  @Test
-  public void testWithGroupBy() throws Exception {
-    String query = "select t1.age.`max` as age, count(*) as cnt, any_value(t1.a) as any_a, any_value(t1.city) as any_city, " +
-        "any_value(f) as any_f, any_value(m) as any_m from  cp.`store/json/test_anyvalue.json` t1 group by t1.age.`max`";
-    testBuilder()
-        .sqlQuery(query)
-        .unOrdered()
-        .baselineColumns("age", "cnt", "any_a", "any_city", "any_f", "any_m")
-        .baselineValues(60l, 2l, TestBuilder.listOf(TestBuilder.mapOf("b", 50l, "c", 30l), TestBuilder.mapOf("b", 70l, "c", 40l)), "San Bruno",
-            TestBuilder.listOf(TestBuilder.mapOf("g", TestBuilder.mapOf("h", TestBuilder.listOf(TestBuilder.mapOf("k", 70l), TestBuilder.mapOf("k", 80l))))),
-            TestBuilder.listOf(TestBuilder.mapOf("n", TestBuilder.listOf(10l, 11l, 12l))))
-        .baselineValues(80l, 3l, TestBuilder.listOf(TestBuilder.mapOf("b", 10l, "c", 15l), TestBuilder.mapOf("b", 20l, "c", 45l)), "Palo Alto",
-            TestBuilder.listOf(TestBuilder.mapOf("g", TestBuilder.mapOf("h", TestBuilder.listOf(TestBuilder.mapOf("k", 10l), TestBuilder.mapOf("k", 20l))))),
-            TestBuilder.listOf(TestBuilder.mapOf("n", TestBuilder.listOf(1l, 2l, 3l))))
-        .go();
+    @Test
```

Comment:
@amansinha100 I have added a testcase for testing multiple(2) batches.

Reply:
Thanks for adding the new test.   LGTM.